Tensile/Tests/extended/local_split_u/igemm_lsu_mfma.yaml

```diff
@@ -371,7 +370,7 @@ BenchmarkProblems:
         - SourceSwap: [0,1]
         #- TransposeLDS: [0,1]
         - GlobalReadVectorWidth: [4,8]
-        - VectorWidth: [2,4]
+        - VectorWidth: [4] # 2
```

Comment:
Could we keep 2 instead of 4?
If we use 4 only, most of MT sizes will not be used because MTWaveTile0 is not multiple of VectorWidth(=4).

Reply:
Fixed